src/renderer/index.js

```diff
@@ -2,6 +2,6 @@ import React from "react";
 import ReactDOM from "react-dom";
 import "./index.css";
 
-import App from "./App";
+import App from "./app";
 
-ReactDOM.render(<App />, App);
+ReactDOM.render(<App />, document.getElementById('app'));
```

Comment:
As I wrote above this is a mistake I did regarding refactoring.
It should be `ReactDOM.render(<App />, app);` because <div id="app"/> is also referenced as global variable. Kinda hacky. I agree that is better to be more explicit, so let's keep your solution 👍